manifest.json

```diff
@@ -3,7 +3,7 @@
     "author": "Stanford GSB",
     "manifest_version": 2,
     "name": "Beyond the Paywall",
-    "version": "0.1.8",
+    "version": "0.1.6",
```

Comment:
```suggestion
    "version": "0.1.8",
```

Reply:
Don't worry about changing version number, we'll do that in a branch before release.

Reply:
OK thanks